manageWebcomponents.js

```diff
@@ -1,28 +1,45 @@
+import api from './apiTemplate.js';
 window.addEventListener('DOMContentLoaded', () => {
     const inputCheckboxes = document.querySelector('input-checkboxes');
     const sectionArticles = document.querySelector('section-articles');
 
-    const sectionArticlesApiSuccess = function(parsedData) {
-        sectionArticles.setData = parsedData.data;
-        sectionArticles.setCurrentCategories = inputCheckboxes.getTags;
-    }
-    const sectionArticlesApiError = function(error) {
-        console.log(error);
-    };
-    sectionArticles.api('http://localhost:3000/bikes', sectionArticlesApiSuccess, sectionArticlesApiError);
-    
-    const inputCheckboxesApiSuccess = function(parsedData) {
-        inputCheckboxes.setData = parsedData.data.map(obj => {
+    const inputCheckboxesApiSuccess = function(categoriesData) {
+        inputCheckboxes.setData = categoriesData.data.map(obj => {
             return { 
                 name: obj.title,
                 description: obj.description
             }
         });
+        
+        const sectionArticlesApiSuccess = function(bikesData) {
+            const theData = bikesData.data.map(bikeObj => {
```

Comment:
You could cleanup line 15 further by destructuring and renaming your parameter names.

```javascript
const sectionArticlesApiSuccess = function({ data: bikesArray }) {
    const theData = bikesArray.map(bikeObj => {
...